text_generation/causal_lm/cpp/continuous_batching/library/src/sequence_group.hpp

```diff
@@ -83,7 +92,15 @@ class Sequence {
     // appends new tokens to a generated part
     void append_token(int64_t token_id, float log_prob) {
         m_cumulative_log_prob += log_prob;
-        m_generated_ids.push_back(token_id);
+        m_generated_ids.push_back(token_id);     
+    }
+
+    GenerationOutput get_last_generation_output(const GenerationConfig& sampling_params) {
+        GenerationOutput output;
+        output.score = get_beam_search_score(sampling_params);
```

Comment:
No action required:
For future PRs from @iefode - we need to set proper score depending on sampling type. E.g. beam search score is not applicable for greedy, we have to use log probs